### PR TITLE
Docs/dashboard dynamic statistics

### PR DESCRIPTION
**Pull request by lucaball**
With this PR I provide documentation for creating custom statistics. With was introduced with #232

### CODE REVIEW

**Comment by lucasballyn** [comment]
Hi @loic425, 

I was not sure if I also had to push the built documentation or not. If so, please let me now. I'll push it.

File: docs/cookbook/dashboard/index.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+.. note::
+    The response of the generate function will be displayed as is in the dashboard.
+    Which means you can return anything, as long as it is a string.
+    Eg. in the CustomerStatistic it is an HTML block which shows you the amount of registered customers.
+
+Order your statistics
+---------------------
+
+Since Symfony 4.4 it is possible to sort your services with a static function called ``getDefaultPriority``.
+Here you need to return an integer to set the weight of the service. Statistics with a higher priority will be displayed first.
```
Comment (loic425) [inline docs/cookbook/dashboard/index.rst]: Yes and I think we can add:
This is why we choose to use a negative value (**-1** for the first element, **-2** for the second...).

File: docs/cookbook/dashboard/index.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+.. code-block:: php
+
+    public static function getDefaultPriority(): int
+    {
+        return -1;
+    }
+
+
+.. note::
```
Comment (loic425) [inline docs/cookbook/dashboard/index.rst]: maybe a warning instead? What do you think?

File: docs/cookbook/dashboard/index.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
+How to create your own statistics
+=================================
+
+The mechanism behind the displaying of statistics relies on tagged services which are supported since Symfony 4.3
+
+Create your own statistic
+-------------------------
+Add a new class to ``/src/Dashboad/Statistic`` and make sure it implement the ``App\Dashboard\Statistics\StatisticInterface``.
```
Comment (loic425) [inline docs/cookbook/dashboard/index.rst]: ``/src/Dashboard/Statistic``
There's a typo on "Dashboard"

**Review by loic425 — CHANGES_REQUESTED** [review]

**Review by loic425 — APPROVED** [review]

**Comment by loic425** [comment]
Thanks @lucasballyn :1st_place_medal:

**Pull request merged**